sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AmqpConstants.java

```diff
@@ -18,4 +18,8 @@ public final class AmqpConstants {
     public static final Symbol STRING_FILTER = Symbol.getSymbol(APACHE + ":selector-filter:string");
 
     static final int AMQP_BATCH_MESSAGE_FORMAT = 0x80013700; // 2147563264L;
+
+    public static final Symbol CLIENT_IDENTIFIER = Symbol.getSymbol(VENDOR + ":client-identifier");
+
+    public static final Symbol CLIENT_RECEIVER_IDENTIFIER = Symbol.getSymbol(VENDOR + ":receiver-name");
```

Comment:
should this be ":receiver-identifier" ?

Reply:
Here is Jesse's doc, https://github.com/Azure/azure-sdk-for-java/issues/22981 we should ":receiver-name" is correct.